lib/dialects/mssql/query-interface.js

```diff
@@ -24,7 +22,7 @@ const _ = require('lodash');
  */
 const removeColumn = function(tableName, attributeName, options) {
   options = options || {};
-  options = _.assign({ raw: true, logging: options.logging}, options);
+  options = Object.assign({}, options);
```

Comment:
@sushantdhiman Would it not be set [here](https://github.com/sequelize/sequelize/blob/master/lib/sequelize.js#L493-L495)